extensions/wikia/UserLogin/js/UserLoginFacebook.js

```diff
@@ -58,7 +58,10 @@
 				self.loginSetup();
 
 				// load when the login dropdown is shown or specific page is loaded
-				$.loadFacebookAPI();
+				$.loadFacebookAPI()
+					.done(function () {
+						$('.fb-loaded').removeClass('hidden');
```

Comment:
Same here, we're querying the DOM in line 79, so we should duplicate that query.